docs/cristin.ttl

```diff
@@ -479,8 +521,9 @@
 :TYPE_BOK_RAPPORT
     a               :cristinfield ;
     rdfs:comment    """
-    A type of book called rapport.
-    """ ;
+        Metadata about a book.
+        This only has a value if the varbeidhovedkatkode has one of the following values OVERSETTELSESARB RAPPORT BOK
```

Comment:
Move this line to the description of the class and in the comment here just point to the class.